sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/WarningCode.java

```diff
@@ -30,7 +30,7 @@
      * @return The corresponding {@link WarningCode}.
      */
     @JsonCreator
```

Comment:
don't need this>

Reply:
Is this a generated model https://github.com/Azure/azure-sdk-for-java/pull/11227/files#diff-2683c3cab1fd268997bc5df017c80afcL3?

Reply:
WarningCode is not a generated model.

if you take a look at what `JsonCreator` annotation's description:
https://fasterxml.github.io/jackson-annotations/javadoc/2.7/com/fasterxml/jackson/annotation/JsonCreator.html

Speaking of what other SDK's behavior of including this annotation. They included it as well.

Reply:
It should be included on the deserialization layer i.e on the service side equivalent model. Don't need it here.